fe/fe-core/src/main/java/org/apache/doris/statistics/ColumnStatistic.java

```diff
@@ -94,10 +95,13 @@ public class ColumnStatistic {
     @SerializedName("updatedTime")
     public final String updatedTime;
 
+    @SerializedName("hotValues")
```

Comment:
please add hotvalues in function: org.apache.doris.statistics.ColumnStatistic#toString